scalameta/contrib/src/test/scala/scala/meta/contrib/ScaladocParserSuite.scala

```diff
@@ -51,6 +51,31 @@ class ScaladocParserSuite extends FunSuite {
     assert(
       parseString(
         s"""
+         // $expectedBody
```

Comment:
The Scaladoc parser will parse them, no problem with that.

If it's unparseable what I should do? `IllegalArgumentException` with a require, return an empty `Seq`, an` `Option` or an `Either`?

Thank you :)